Packs/QualysFIM/Integrations/Qualys/QualysFIM.py

```diff
@@ -0,0 +1,643 @@
+            outputs.append(item_dictionary)
+
+    readable_output = tableToMarkdown(name=f'Listed {len(outputs)} Incidents:', t=outputs,
+                                      headers=table_headers, removeNull=True)
+    return CommandResults(outputs_prefix='QualysFIM.Incident',
+                          outputs_key_field='id',
+                          raw_response=raw_response,
+                          outputs=raw_outputs,
+                          readable_output=readable_output)
+
+
+def list_incident_events_command(client: Client, args: dict):
+    """
+    Get a specific incident's events by incident ID.
+
+    Args:
+        client (Client): Qualys FIM API client.
+        args (dict): All command arguments.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    params = remove_empty_elements({'filter': args.get('filter', None),
+                                    'pageNumber': args.get('page_number', None),
+                                    'pageSize': args.get('limit', None),
+                                    'attributes': args.get('attributes', None)})
+    raw_response = client.get_incident_events(args.get('incident_id', None), params)
+    table_headers = ['id', 'name', 'severity', 'action', 'type', 'dateTime']
+    outputs = []
+    raw_outputs = []
+    if raw_response:
+        for item in raw_response:
+            data = item.get('data', None)
+            if data:
+                raw_outputs.append(data)
+                item_dictionary = create_event_or_incident_output(data, table_headers)
+                date_time = item_dictionary.get('dateTime', None)
+                if date_time:
+                    date_time = datetime.strptime(date_time,
+                                                  DATETIME_FORMAT).strftime(TABLE_DATETIME_FORMAT)
+                    item_dictionary['dateTime'] = date_time
+                outputs.append(item_dictionary)
+    readable_output = tableToMarkdown(name=f'Listed {len(outputs)} Events From Incident:',
+                                      t=outputs, headers=table_headers, removeNull=True)
+    return CommandResults(outputs_prefix='QualysFIM.Event',
+                          outputs_key_field='id',
+                          raw_response=raw_response,
+                          outputs=raw_outputs,
+                          readable_output=readable_output)
+
+
+def create_incident_command(client: Client, args: dict):
+    """
+    Create a new incident.
+
+    Args:
+        client (Client): Qualys FIM API client.
+        args (dict): All command arguments.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    if len(str(args.get('name', None))) > 128:
+        raise ValueError('Name is limited to 128 characters, please shorten name.')
+    if args.get('filters'):
+        filters = (args.get('filters'))
```

Comment:
```suggestion
        filters = f'({args.get('filters')}')
```

If I understand it correctly, you just wanted it to be surrounded with `()`, so it is better that way.